src/functions/keycloak/find-user/handler.ts

```diff
@@ -1,19 +1,10 @@
 import type { APIGatewayProxyEvent, APIGatewayProxyHandler } from "aws-lambda";
 import { CreateKeycloakApiService } from "src/database/services/keycloak-api-service";
 import { middyfy } from "src/libs/lambda";
-
+import type { User } from "src/types/keycloak/user";
 /**
- * Response schema for lambda
+ * FIXME: At this moment in KeyCloak its called: severa-user-id (string), in here its severaGuid (string);
```

Comment:
If this relates to the severaGuid attribute in filteredUser then I'd suggest putting the comment directly above the attribute. However if the attribute name is defined in the User type, then I'd suggest putting this comment there- unless there is a reason not to?